src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -48,13 +56,32 @@ export const VenueAdminPage: React.FC = () => {
     );
   }
 
-  const isIframeVenue = IFRAME_TEMPLATES.includes(venue.template);
-
   return (
-    <>
-      <h4 className="admin-page-title">You are editing venue: {venue.name}</h4>
-      <BannerAdmin venueId={venueId} venue={venue} />
-      {isIframeVenue && <IframeAdmin venueId={venueId} venue={venue} />}
-    </>
+    <WithNavigationBar>
+      <div className="VenueAdminPage">
+        <h4 className="VenueAdminPage__title">
+          Current Announcement in Space Title
```

Comment:
This should be using the venue name instead of `Space Title`. Take a look at the issue's images and you'll see that `Space Title` is the name of the venue in the design.

Reply:
fixed 947713e